tests/unit/openwrt/golden/ipq806x/root/.ssh/authorized_keys

```diff
@@ -1,4 +1,5 @@
-ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQDrMp0YYH1epC8A3dci7gSPB/Prko4/ULY74kD/3kSrF/4FDfsfhTVo1on1SN42fdBbbUFc0rwoy1qhFETs+MmJIplmhU5d72nStR/jDjFbshKEtTFrg02uatVbq/xDCsdgq/uJ7UJaa5a8m5XEO/rqziPWipDe5qCj3V8pX/+oCBccnqtnTvY9WsxJioOwwSu45NbW7Bhg2hOHFCwj0qcGT7Gfa6EljAypOcxaGKH/aFsHM239OedXP9QAr7XiJ5px6EFvVSqkX3tqxpw6/7KnMrSrkC4OXacxIZyibvy0IdQm0fVljx876Sf5a7n9vZMvWhEEgxf3IH2b2zZzPtuR9HwoqTqnN37kBIeh0WpFwmkRLxgcueZdRv9PxZ2pIPrxRyMLNj4750n8GAmXzwaDv4fNUr6vMosj2umnxyWGFw48tq8Vn6b75MeadI2Tu07occJRKnYN+5W2OVWhwi17f2SZT9NdHBIJJYu1GuzG6e4EwHzTz+iqxRtjTWxrkDCcopKmwyc42seN/JdozzEaATAY0h13XhM1k26vNkafhCkWQcTaTpZpjTc1Voz0jPqQajjP4d1krHs8377lO1ZZ5ZADXFykgvGzflf93WREqo9MODlbXbkxpkvqiyBIZM/28IR/EsnHJhYvY5ucEAMovAyp1tu567QPZNJF6XRsKw==
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQDB3ATAbZj/tJXZsRnlpvAXphtY487R4EgDzBiVAf3BQxDzBnJ2T8KFTUUT/SKvq98erW3sCCuEFTYM4CVEm1uDDqhVfWuRbihx8tECZf3AYix3jslZrRUOBkHENxfSIqIwdtwKMuhWUjOhUZY396HMEJ4Qgo+KajAxFC1o2qSBaHN5gsCdZBUd21zJ1+eZHanv6WvwbXQAd4SPjEr61FV4Nw7LWp/theyMVbLYQgpj8kyQK2O2aF08Ts5R7kVSVmZujlUtI4QhObucOdbpfHEc1W0mvrQ5RxNXLElw22OcypEY4/K7eCcb77PHpdBbqqwChBQjx19DBnz0jSuWSCMDwBd1Z68P227IGM8R6L5HLJJpGNWP1jiDvFoCs2YqNlMrMgOLwUnclKWrhppdHcOTvatgDE238XFjQrod2uXORVE3egcWYFqC2L2qilCCBdC3Mf6TRbREZlQ5G79QdLmBBzB8u0DnSFOcQNRFRitbQKUsvdNUeM+GzYciWwPc5tNhEWXlAoSoHYFgDNjzJrU67IuSTT0GJ10Ijs/gCJLX5ctfra3uvgkeXTI49SdD4PSaXYvAGotb7clB6IHb/zwdQFdLtpo54FDQAEOWM00soLd0F2aj2gTmZ/kI/KpBz7ZsJvN35SSudQOZk+PTECWgfQuJHJh5502UsuddgDxi7w==
 ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBBjjcUJLTENGrV6K/nrPOswcBVMMuS4sLSs0UyTRw8wU87PDUzJz8Ht2SgHqeEQJdRm1+b6iLsx2uKOf+/pU8qE= root@kiev.delong.com
 ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQDBP0XP83nOtxnsGJ7AG+FuT7dbAEO8BrIHlPbhCGhKjFKcq8g6LWa0htx3iwHJB2VfasIVSqWAC7SwX81XfSTwpvNSNVXvrIcMNdYOpEBHx5W94iVP2ooVVsGIeK/M+Eu9toiHebtVFT45hQ8VO5DwqbG6xPgEp1PmLsPhyBwP0LyC8Q4tA2ttftC6/fJa6fa1G54c7D/euIhZnyM/ovcpYfNYTlarX/yLJoYaOEk5ByYJPYnpQtm2GQX8CdT/U14nDtJ/VjzW8kP/R6+ZsmGpxBrWp2Mn03LjL4CzTSN3cbE2CAKKSjkIcaaoohRZS+JXnxm5+TXF/6QmN0MPU3SSThKWIzOhzfMrLqOu6hsw16x9lF1eJreUZr/WysWOswM4WNoffBv6mKvVsMR8Z8tgERR8+sIxUaoVOfChR9wRoD5lcNnVxrC6rcF+rBxeY4JiPS3zpyamwteyR0n2YspT9Dl0MlZb4cWml2aBnvYk0TmoCC3eGs+2/wssL4s+64TiFC8WkK7U7RW0eYODwPJQbzNGZBjcsVlKD4whs5cNGlgjkZJzTdFt49EwIKmFnSd9Ly/NY3zhByFS/MN3Mj9I2AqMRZ/i4iT72gATE9aLle1xViGxjK7bHmHT2H1yqQhJ5bMcorpDzSZyTkyBTDDnOgh8tM+tBG204kYywQW/RQ==
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDrMBEZC9WB9dJMGoev5b+P8I5UtcQLLpoBfKDrEF8bHdQ1AcdpCMtEnTVNmqVMzq8+gRR2CzbqI1Ue7KteFtZX/YtKIEEyL6D5kzhWyQrybl35nxA5VIKJrNszHdXDb/MGMwTMfWV1/VEka4UnCwXpgPH/AXQ/abhmFFDAJnc+PmqXPX3fHA4J5fpl/6CkT7JLKGMpsI7LCtU6cohtnXZrDHqFmPa6T+nHv2cT2QF6eSIPuMX7sLth6QI92yTam8+QUAeMFzEZOcmb6W8CNGio13ezn8ljQ6lB5uZLHhnnuxG9z6RJZugMzU9UxpkHRSKUVX9+qRGf1ntiPKE5Pb4x bibayoff@gmail.com
```

Comment:
the automation is already paying off, apparently we already had drift between the two sets of keys. so that's nice.